packages/@react-aria/interactions/test/useFocusVisible.test.js

```diff
@@ -50,3 +62,17 @@ describe('useFocusVisible', function () {
     expect(el.textContent).toBe('example');
   });
 });
+
+describe('useFocusEmitter', function () {
+  it('emits on modality change', function () {
+    let fnMock = jest.fn();
+    renderHook(() => useFocusVisibleListener(fnMock, []), {isTextInput: true});
+    expect(fnMock).toHaveBeenCalledTimes(0);
+    fireEvent.keyDown(document.body, {key: 'Escape'});
```

Comment:
i know it only relies on the keydown and mouseDown, but for completeness, lets include the keyup and mouse up as well